keyboards/keebzdotnet/wazowski/info.json

```diff
@@ -1,25 +1,25 @@
 {
     "keyboard_name": "wazowski",
-    "url": "",
+    "url": "https://www.keebz.net/shop/wazoski-pcb",
     "maintainer": "keebzdotnet",
     "width": 9,
     "height": 5,
     "layouts": {
         "LAYOUT": {
             "layout": [
-                {"label":"k00", "x":0, "y":3},
-                {"label":"k01", "x":1, "y":0},
-                {"label":"k02", "x":1, "y":4},
-                {"label":"k03", "x":2, "y":3},
+                {"label":"k01", "x":1, "y":1},
```

Comment:
The ordering here is kinda strange (testing in Configurator Ctrl+Shift+I with fast input enabled). Is that intentional?

Reply:
yeah its intentional and looks really weird but this is working and live already for people to use.